tests/test_issues/test_issue_1238.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+
+from linkml_runtime import SchemaView
+
+META_URL = "https://raw.githubusercontent.com/linkml/linkml-model/main/linkml_model/model/schema/meta.yaml"
```

Comment:
tests should be self contained and have no network dependencies

Additionally this test will break as soon as the metamodel changes and a new metaslot is added to ClassDefinition

Reply:
OK, I'm adding to this now but will ultimately remove the test.

If I did want to retain this test, is the metamodel available from linkml/linkml?

Also, what does it mean to add a metaslot to `ClassDefinition`, as opposed to a slot? Does "metaslot" emphasize that it's a slot from the metamodel, as opposed to a user schema? Is metaslot defined anywhere?

Reply:
Thanks for the review

Reply:
This was a starting point for 
- #1239

Reply:
> OK, I'm adding to this now but will ultimately remove the test.

I see. I encourage PRs for test cases but if the goal is to do entirely local testing best not to make a PR

> Also, what does it mean to add a metaslot to ClassDefinition, as opposed to a slot? Does "metaslot" emphasize that it's a slot from the metamodel, as opposed to a user schema? Is metaslot defined anywhere?

a metaslot is a slot in the metamodel; see 
https://linkml.io/linkml-model/docs/specification/03schemas/#schema-basics